CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- Buttons StreamBlock for WhatsApp messages
+
+### Changed
+- Migrated next_prompt on WhatsApp messages to buttons
```

Comment:
If we're deprecating `next_prompt` instead of removing it, do we still want this migration?

Reply:
Yes I think so, it will allow people to easily move over to using the new format, instead of content admins having to manually go through all the content and add buttons.

Reply:
So existing `next_prompt`s will be migrated, but then any new ones (added or imported) will need to be manually changed over at some point before `next_prompt` goes away? Is it worth adding a management command or something to do that?

Reply:
Migrating at this point, when we know `buttons` is empty, is easy to do. Doing it later on is much more difficult, eg. what do you do if there are already 3 buttons, and will require human intervention, so I'm happy leaving that for now, and working through any specific use cases (and maybe creating a management command at that point once we know more)